scheds/rust/scx_layered/src/main.rs

```diff
@@ -1448,42 +1462,38 @@ impl<'a, 'b> Scheduler<'a, 'b> {
 
         // Other stuff.
         let proc_reader = procfs::ProcReader::new();
-        let sys_stats = Arc::new(Mutex::new(SysStats::default()));
 
-        let mut sched = Self {
-            struct_ops: None,
+        // XXX If we try to refresh the cpumasks here before attaching, we
```

Comment:
good call on this, I think I've seen this while testing some of the topology awareness